frappe_pwa/frappe_pwa/doctype/web_app_manifest/web_app_manifest.py

```diff
@@ -7,11 +7,19 @@
 
 from frappe.model.document import Document
 
+
 class WebAppManifest(Document):
 
-	def on_update(self):
-		"""clear cache"""
-		frappe.clear_cache(user = 'Guest')
+    def on_update(self):
+        """clear cache"""
+        frappe.clear_cache(user='Guest')
+
+        from frappe.website.render import clear_cache
+        clear_cache()
 
-		from frappe.website.render import clear_cache
-		clear_cache()
+    def configure_pwa(self):
+        ws = frappe.get_doc('Website Settings')
+        ws.head_html = '''<link href="/assets/frappe_pwa/manifest.json" rel="manifest">'''
+        ws.save()
+        self.is_asked = 1
+        self.save()
```

Comment:
@AminovE99 If previous "_smart_" behavior is implemented, this is not needed and can be removed.